Terminal/Terminal.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity >= 0.6.0;
+
+interface ITerminal {
+	
+	function inputStr(uint32 answerId, string prompt, bool multiline) external returns (string value);
+	function inputInt (uint32 answerId, string prompt) external returns (int256 value);
+	function inputUint(uint32 answerId, string prompt) external returns (uint256 value);
+	function inputTons(uint32 answerId, string prompt) external returns (uint128 value);
+	function inputBoolean(uint32 answerId, string prompt) external returns (bool value);
+	function print(uint32 answerId, string message) external;
+	function printf(uint32 answerId, string fmt, TvmCell fargs) external;
```

Comment:
This method should be implemented on dengine, and here we should leave only `print`